lib/compress/index.js

```diff
@@ -4994,8 +4994,8 @@ def_optimize(AST_Call, function(self, compressor) {
             }
             break;
           case "Symbol":
-            if (compressor.option("unsafe_symbols"))
-                return make_node(AST_SymbolRef, self, exp).optimize(compressor);
+            if (self.args.length == 1 && compressor.option("unsafe_symbols"))
```

Comment:
Here you also want to check that self.args[0] is an instance of AST_String

Reply:
Fixed, thanks!